src/platform/nrfconnect/OTAImageProcessorImpl.h

```diff
@@ -51,14 +51,14 @@ class OTAImageProcessorImpl : public OTAImageProcessorInterface
     CHIP_ERROR Finalize() override;
     CHIP_ERROR Abort() override;
     CHIP_ERROR Apply() override;
-    CHIP_ERROR ProcessBlock(ByteSpan & block) override;
+    CHIP_ERROR ProcessBlock(ByteSpan & aBlock) override;
     bool IsFirstImageRun() override;
     CHIP_ERROR ConfirmCurrentImage() override;
 
 private:
     CHIP_ERROR PrepareDownloadImpl();
-    CHIP_ERROR ProcessHeader(ByteSpan & block);
-    CHIP_ERROR SwitchToNextImage(ByteSpan & aRemainingData);
+    CHIP_ERROR ProcessHeader(ByteSpan & aBlock);
```

Comment:
nit: Have you checked if `const` also can be applied in case of `ProcessHeader` and `ProcessBlock`?

Reply:
In `ProcessBlock` method this can't be applied due to this method overrides the virtual OTAImageProgress's method, and these types must be coherent. In `ProcessHeader` method the ByteSpan is edited in `AccumulateAndDecode` method so it can't be const here. Only in my `SwitchToNextImage` method it has been applied because there I used this Bytespan to fill up DFU's data buffer.